demo/ez-browse.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>ez-browse demo</title>
+
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+    <link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">
+    <link rel="import" href="../../iron-demo-helpers/demo-snippet.html">
+    <link rel="import" href="../ez-browse.html">
+
+    <style is="custom-style" include="demo-pages-shared-styles">
+        ez-browse.demoTest:after {
```

Comment:
`demoTest` class is not set anywhere, so this is not applied so this can probably be removed. also, we don't have any shared style, you can remove `<link rel="import" href="../../iron-demo-helpers/demo-pages-shared-styles.html">` as well